src/commands/Minion/tithefarm.ts

```diff
@@ -0,0 +1,97 @@
+import { CommandStore, KlasaMessage, KlasaUser } from 'klasa';
+
+import { BotCommand } from '../../lib/BotCommand';
+import { Time, Activity, Tasks } from '../../lib/constants';
+import { formatDuration, rand } from '../../lib/util';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { TitheFarmActivityTaskOptions } from '../../lib/types/minions';
+import { minionNotBusy, requiresMinion } from '../../lib/minions/decorators';
+import { SkillsEnum } from '../../lib/skilling/types';
+import { MinigameIDsEnum } from '../../lib/minions/data/minigames';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+import hasArrayOfItemsEquipped from '../../lib/gear/functions/hasArrayOfItemsEquipped';
+import itemID from '../../lib/util/itemID';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			oneAtTime: true,
+			altProtection: true
+		});
+	}
+
+	determineDuration(user: KlasaUser): [number, string] {
+		let baseTime = Time.Second * 1538;
+		let nonGracefulTimeAddition = Time.Second * 123;
+
+		let boostStr = '';
```

Comment:
btw, here - and everywhere else you do similar stuff, you should use an array. This way you can easily format the array in any ways you want. e.g. `boosts.join(', ')`